.github/workflows/update-k8s-manifest.py

```diff
@@ -0,0 +1,56 @@
+from fileinput import close
```

Comment:
I would be interested in having tests for this code.... which would require the actual caller setting a flag to say "actually run me" versus "load for for testing".  This won't block the merge, but something to think about.

Reply:
I'll think about how that can be structured.